packages/codegen/package.json

```diff
@@ -16,13 +16,13 @@
   "scripts": {
     "tsc": "tsc",
     "lint": "eslint",
-    "copy-hygen": "cp -r src/codegen/.hygen dist/src/codegen/",
+    "copy-hygen": "rm -rf dist/src/codegen/.hygen && cp -r src/codegen/.hygen dist/src/codegen/",
```

Comment:
when we copy the hygen templates, we need to first purge the dir. otherwise deleted templates will still appear in the final result.